temba/classifiers/views.py

```diff
@@ -99,3 +114,60 @@ def get_context_data(self, **kwargs):
             context = super().get_context_data(**kwargs)
             context["classifier_types"] = Classifier.get_types()
             return context
+
+    class Train(OrgObjPermsMixin, SmartUpdateView):
+        slug_url_kwarg = "uuid"
+
+        class Form(forms.Form):
+            file = forms.FileField(validators=[FileExtensionValidator(allowed_extensions=("csv",))])
+            languages = forms.MultipleChoiceField(
```

Comment:
Let's rename this field to not conflict with the languages imported at the beginning of this file https://github.com/greatnonprofits-nfp/rapidpro/pull/340/files#diff-9c465579910a52b0d5bc483683fb3b2d46ae9f8ef5f4b755a50a16335f39819aR12